src/args.py

```diff
@@ -21,13 +31,16 @@
 a = [7, 6, 5, 4]
 
 # What thing do you have to add to make this work?
-print(f2(a))    # Should print 22
+print(f2(*a))    # Should print 22
```

Comment:
I'm not sure what this asterisk is doing. Does this print 22 as expected? I would add a check for seeing if nums is a list in the f2 function definition and sum the parts of the list by index if it is.

Reply:
It seems to pass the values of the list as separate arguments. And it does return 22 as expected, but I see the value of adding a check in the function versus just spreading in the values.